source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRViewPrivate.h

```diff
@@ -11,15 +11,13 @@
 
 @interface ACRView()
 
-- (void)loadImage:(NSString *)url key:(NSString *)key;
+- (void)loadImage:(NSString *)url key:(NSString *)key resolver:(NSObject<ACOIResourceResolver> *)resolver;
```

Comment:
This method is never implemented or called?

Reply:
thanks; somehow i missed it.

---
In reply to: [202426851](https://github.com/Microsoft/AdaptiveCards/pull/1702#discussion_r202426851) [](ancestors = 202426851)